postgres/init/20-portal.sql

```diff
@@ -186,8 +186,13 @@ CREATE TABLE portal.apps (
     name character varying(255),
     date_released timestamp without time zone,
     thumbnail_url character varying(255),
+    thumbnail_url1 character varying(255),
```

Comment:
I would prefer if we made a separate table that can have unlimited amounts of thumbnail urls assigned to an app. Maybe @ntruchsess has suggestions as well.

Reply:
@wira-mms what would be the proposal for the table? I would definitely want to ensure that we can distinguish between leadPicture and detailPicture

Reply:
@jjeroch For example, we could keep thumbnail_url (lead picture) inside apps and make a new table app_assigned_detail_pictures with id (uuid), appId (uuid) and picture_url (varchar). That way you can only have one lead picture and as many detail pictures as you want without having those weirdly named and possibly empty fields.

Reply:
@wira-mms for me fine. I dont have a strong passion for the one or the other way, so would directly do it like that. Will push the commit in the next 20min

Reply:
@wira-mms changes done